Debian/8/Makefile

```diff
@@ -127,5 +127,5 @@ dist: guide content
 clean:
 	rm -f $(OUT)/*.xml $(OUT)/*.html $(OUT)/*.xhtml $(OUT)/*.pdf  $(OUT)/*.spec $(OUT)/*.tar $(OUT)/*.gz $(OUT)/*.ini $(OUT)/*.csv
 	rm -rf $(DIST)/content $(DIST)/guide
-	$(SHARED)/utils/build-remediations.py --output ./templates clean
+	rm -rf ./templates/output
```

Comment:
I'd rather the templates/output directory never get created in the first place, here and in subsequent Makefile clean targets.